### PR TITLE
Create a pop-up of Provide help form for volunteers 

### LINKED ISSUE #40: Create a pop-up dialog Provide Help Form for volunteers
**User story**
As a volunteer, I want to be able to provide details of my skillset and what kind of help I can provide so that business owners who need help out can find the right skillset of volunteers.

**Acceptance criteria**
  * Create `VolunteerForm` using Dialog API available on Material UI (should be similar as #10)
  * Add input/text area/select, etc. so that user can provide details of their skillset:
  * Name (text field)
  * Location (Google Maps API -- to be completed later)
  * Description (multi-line text field)
      - Skillset details: multi-line text field (portfolio or resume)
      - a set of tags to associate skillset with the right keywords
  * Ensure that the basic layout of the form aligns with the mockup

**Note**
* Storing the information is not required at this stage

### PR DESCRIPTION
**Pull request by anhyesun**
#### [Create a pop-up of Provide help form for volunteers ](#40)

**What and how**
* Replicated from business owner's request pop-up 
* Split the button action from request pop-up
* Note: Functionality of Tags will be applied later
- [x] Ran lint: fix before submitting the code

**Screenshots**
volune

### CODE REVIEW

**Comment by yehee** [comment]
Would it be possible that the `Portfolio or Resume Link` be either a single-line text input or an actual file upload?

File: client/src/components/Form/VolunteerForm.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    Actions,
+    Content,
+    ContentText,
+    Input,
+    Text,
+} from './components';
+import Tags from '../Tags/Tags';
+import classes from '../../modules/form.module.css';
+
+const styles = {
```
Comment (yehee) [inline client/src/components/Form/VolunteerForm.js]: Seems like we don't need this?

File: client/src/components/Form/VolunteerForm.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+const VolunteerForm = ({ open, handleClose }) => {
+    const theme = useTheme();
+    const fullScreen = useMediaQuery(theme.breakpoints.down('sm'));
+
+    return (
+        <Dialog
+            fullScreen={fullScreen}
+            open={open}
+            aria-labelledby="request-form"
```
Comment (yehee) [inline client/src/components/Form/VolunteerForm.js]: We could use `volunteer-form` instead

File: client/src/components/Form/VolunteerForm.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        >
+            <Title id="request-form" disableTypography>Volunteer Registration Form</Title>
```
Comment (yehee) [inline client/src/components/Form/VolunteerForm.js]: Same here!

**Comment by anhyesun** [comment]
- [x] Reflected review comments on the commit

**Review by yehee — APPROVED** [review]
LGTM

**Pull request merged**